keylime/tenant.py

```diff
@@ -871,13 +918,19 @@ def do_reglist(self) -> Optional[Dict[str, Any]]:
         if not self.registrar_ip or not self.registrar_port:
             raise UserError("registrar_ip and registrar_port have both to be set in the configuration")
 
+        self.set_full_id_str()
+
         response = registrar_client.doRegistrarList(
             self.registrar_ip, self.registrar_port, tls_context=self.tls_context
         )
 
+        # Marked for deletion (need to modify the code on CI tests)
         logger.info(
-            "From registrar %s port %s retrieved %s", self.registrar_ip, self.registrar_port, json.dumps(response)
+            "From registrar %s port %s retrieved %s\n", self.registrar_ip, self.registrar_port, json.dumps(response)
         )
+
+        logger.info("Agent list from %s retrieved: \n%s", self.registrar_fid_str, json.dumps(response["results"]))
```

Comment:
The rest of the json.dumps are not preceded with \n. Should we remove this, or add it into the rest?

Reply:
so, the "trick" here is let the json string start on a new line, and then use `grep ^{ | jq .` to capture and (json) process the output